nca/nca_cli.py

```diff
@@ -187,8 +187,11 @@ def run_args(args):  # noqa: C901
 
     if args.explain is not None and args.optimized_run == 'true':
         output_config['explain'] = args.explain
-        ExplTracker().activate()
-        ExplTracker().set_endpoints(output_config.outputEndpoints)
+        ExplTracker().activate(output_config.outputEndpoints)
+
+    if args.output_format == 'html':
```

Comment:
stop the run if optimized flag is not true?

Reply:
fixed